docs/mkdocs.en.yml

```diff
@@ -10,16 +10,26 @@ nav:
   - Cost: cost.md
   - Architecture Overview: architecture.md
   - Security: security.md
-  - Supported regions: regions.md
-  - AWS CloudFormation template: template.md
+  - Supported Regions: regions.md
+  - AWS CloudFormation Template: template.md
   - Automated Deployment: deployment.md
   - Workshop: 
     - Overview: workshop/background.md
     - Molecular Unfolding: 
-      - Background: workshop/a-molecular-unfolding/molecular-unfolding.md
-      - Notebook Experiment: workshop/a-molecular-unfolding/notebook-experiment.md
-      - Batch Evaluation: workshop/a-molecular-unfolding/batch-evaluation.md
-      - Evaluate Your Own Model: workshop/a-molecular-unfolding/evaluate-your-own-model.md
+      # - Background: workshop/a-molecular-unfolding/molecular-unfolding.md
```

Comment:
if it's useless, removing commented lines

Reply:
done